### PR TITLE
[Enhancement] Change Quiz Layout of Drag Items

### PR DESCRIPTION
**Pull request by schultek**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.

### Motivation and Context

The Drag Items in Quiz Drag&Drop Questions are spaced wrongly.

### Description

- Fix spacing of items
- Fix overflow for long text items

### Steps for Testing

1. Log in to Artemis
2. Navigate to a quiz exercise with d&d questions.
3. Check the layout for various screen sizes and number of items (text & image items).

### Screenshots

Bildschirmfoto 2020-07-21 um 17 43 36
Bildschirmfoto 2020-07-21 um 17 43 52

### CODE REVIEW

File: src/main/webapp/app/exercises/programming/shared/code-editor/actions/code-editor-resolve-conflict-modal.component.ts [hunk 1]
```diff
@@ -11,6 +11,8 @@ export class CodeEditorResolveConflictModalComponent {
 
     shouldReset: EventEmitter<void> = new EventEmitter<void>();
 
+    shouldReset: EventEmitter<void> = new EventEmitter<void>();
```
Comment (krusche) [inline src/main/webapp/app/exercises/programming/shared/code-editor/actions/code-editor-resolve-conflict-modal.component.ts:14]: please remove this change as it is not related to this PR

File: src/main/webapp/app/exercises/programming/shared/code-editor/actions/code-editor-resolve-conflict-modal.component.ts [hunk 2]
```diff
@@ -11,6 +11,8 @@ export class CodeEditorResolveConflictModalComponent {
...
+
```
Comment (schultek) [inline src/main/webapp/app/exercises/programming/shared/code-editor/actions/code-editor-resolve-conflict-modal.component.ts:15]:
```suggestion
```
